src/css/variables.scss

```
@@ -0,0 +1,359 @@
+  --src-gray-900: #171717;
+
+  // Named text roles from the brand's dark-mode guidance (secondary is
+  // --src-gray-600 already).
+  --src-text-muted: #939598; // muted/tertiary on dark
+  --src-text-subtitle: #adadad; // subtitle on dark
+
+  // Layout
+  // --root-wrapper-width is set on body below, with --doc-sidebar-width.
+  --menu-indicator-color: var(--src-gray-300);
+  --docs-title-border: var(--src-gray-200);
+  // --doc-sidebar-width is set on body below, not here.
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)